src/token/ERC721/ERC721Enumerable/ERC721EnumerableBurnFacet.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.30;
+
+import {LibERC721 as LibERC721Enumerable} from "./LibERC721Enumerable.sol";
```

Comment:
Each facet is self-contained and is not supposed to import anything. Please read the documentation here: https://compose.diamonds/docs/design/